bindings/redis/metadata.yaml

```diff
@@ -193,3 +193,12 @@ metadata:
       "-1" disables idle timeout check.
     default: "5m"
     example: "10m"
+  - name: useEntraID
```

Comment:
This should be a separate authentication profile. See how it's done in Postgres for example (and same on other metadata.yaml's)

Reply:
Ah right - let me take a look. I forgot about that.